migration/migration/015-tracing-compression.sql

```diff
@@ -1,67 +1,341 @@
+-- Due to incompatible change of trace_id type we need to drop tables and delete data
+DROP TABLE _ps_trace.link;
+DROP TABLE _ps_trace.event;
+DROP TABLE _ps_trace.span;
 
--- Fix compression on trace tables
+TRUNCATE TABLE _ps_trace.tag;
+DELETE FROM _ps_trace.tag_key WHERE id >= 1000;
+ANALYZE _ps_trace.tag_key;
+REINDEX TABLE _ps_trace.tag_key;
+PERFORM setval('_ps_trace.tag_key_id_seq', 1000);
+TRUNCATE TABLE _ps_trace.operation;
+ANALYZE _ps_trace.operation;
+TRUNCATE TABLE _ps_trace.instrumentation_lib CASCADE;
+ANALYZE _ps_trace.instrumentation_lib;
+TRUNCATE TABLE _ps_trace.schema_url CASCADE;
+ANALYZE _ps_trace.schema_url;
+
+CALL _prom_catalog.execute_everywhere('tracing_types', $ee$ DO $$ BEGIN
```

Comment:
As discussed on Slack, it's not clear to me that we should be adding this type in an `execute_everywhere` block. I think instead we should be running something like `CREATE EXTENSION promscale`.

Reply:
That is a separate design decision and I don't think we should be conflating that with a newly added custom type. At the moment we are not installing Promscale extension on data nodes.

Reply:
fyi `execute_everywhere` mechanism is already in use in many different places